test/smoke/testApps/TraceLog4j1_2/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/TraceLog4j12Test.java

```diff
@@ -78,7 +78,7 @@ public void testTraceLog4j12() throws Exception {
     assertEquals("This is log4j1.2 fatal.", md3.getMessage());
     assertEquals(SeverityLevel.Critical, md3.getSeverityLevel());
     assertEquals("Logger", md3.getProperties().get("SourceType"));
-    assertEquals("FATAL", md3.getProperties().get("LoggingLevel"));
+    assertEquals("ERROR", md3.getProperties().get("LoggingLevel"));
```

Comment:
we can change this back now with latest change
```suggestion
    assertEquals("FATAL", md3.getProperties().get("LoggingLevel"));
```